source/includes/_payment_routing.md

```diff
@@ -288,7 +288,7 @@ Endpoint:
 | sender_email | String     |  FALSE | - | Email of sender |
 | receive_amount | Numeric     |  TRUE | - | The amount of a transaction to be paid, min. amount is 10000 |
 | va_display_name | String     |  FALSE | Partner's brand name | Display name for VA that will be displayed once user do inquiry. If empty VA name will be set using partner brand name |
-| trx_expiration_time | Date string; yyyy-mm-dd hh:mm:ss format     |  FALSE | 24 hours | Set expiration time of transaction. If empty use default 24h.  Min exp time is 1 hour for non-QRIS transactions. For QRIS transactions (`need_frontend: false, list_enable_payment_method: "QRIS", list_enable_sof: "QRIS"`), this value is limited to 1 minute - 1 hour after the request is sent, the QRIS validity period is dynamic within the aforementioned limit. |
+| trx_expiration_time | Date string; yyyy-mm-dd hh:mm:ss format (UTC+7)    |  FALSE | 24 hours | Set expiration time of transaction. If non QRIS and empty use default 24h, but if using QRIS and empty use default 30m.  Min exp time is 1 hour for non-QRIS transactions. For QRIS transactions (`need_frontend: false, list_enable_payment_method: "QRIS", list_enable_sof: "QRIS"`), this value is limited to 1 minute - 1 hour after the request is sent, the QRIS validity period is dynamic within the aforementioned limit. |
```

Comment:
Set expiration time of transaction. For QRIS transactions (need_frontend: false, list_enable_payment_method: "QRIS", list_enable_sof: "QRIS"), the valid value is between 1 minute and 1 hour (default to 30 minutes if empty) after the request is sent. For others, the minimum value is 1 hour  (default to 24 hours if empty) after the request is sent.

Gini gmn?

Reply:
Oh aku pindahin default valuesnya ke kolom default aja, harusnya lebih enak wordingnya